lib/install/validate/storage.go

```diff
@@ -31,39 +32,34 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 	defer trace.End(trace.Begin(""))
 
 	// Image Store
-	imageDSpath, ds, err := v.DatastoreHelper(ctx, input.ImageDatastoreName)
+	imageDSpath, ds, err := v.DatastoreHelper(ctx, input.ImageDatastoreName, "", "--image-datastore")
 	v.NoteIssue(err)
-	if ds == nil {
-		log.Errorf("Datastore for images is nil")
-		return
+	if ds != nil {
```

Comment:
In vic-machine we prefer to find all issues with the user's input in one pass so they don't have to fix one thing at a time then rerun the command. Here I wanted to check volume stores also even if the image store is incorrect. Can anyone suggest if there is a better way?